com.unity.render-pipelines.high-definition/Runtime/Deprecated.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    /// <summary>Deprecated DensityVolume</summary>
+    [Obsolete("DensityVolume has been deprecated (UnityUpgradable) -> LocalVolumetricFog", false)]
```

Comment:
We actually have the VolumeComponentDeprecated attribute that could be better for this as it also avoid the deprecated volume type to appear in the creation list when adding component overrides.

Reply:
`VolumeComponentDeprecated` is for clases that inherit from `VolumeComponent`. As Density Volume is not from this framework it can not be added using the `Add Override` on a `VolumeComponentList`. 

![image](https://user-images.githubusercontent.com/76204843/111285894-e94bfe80-8641-11eb-867c-5dc2e0bc616a.png)